### ISSUE TITLE
Proposal: add build instructions for specific versions of dart-sdk

### ISSUE THREAD

**Issue by crunchex** [dart-lang/sdk#24099]
I'm having trouble figuring out the exact instructions to follow for building a specific version of dart-sdk. The instructions in the wiki only give you the basic `gclient config` and `gclient sync` commands, which only get you the bleeding edge. I've found that you can specify a revision number with `gclient sync`, but then other things go wrong, like the makefiles don't get generated (this is what happened to me when trying to set up a build for 1.10.1).

And the reason why I can't just download one of the archives off dartlang.org is because I'm running into glibc version issues on Debian, and the only solution I've found is to build the dart-sdk from source on my target system.

**Comment by whesse** [comment]
The default instructions make a .gclient file in the base directory with the line
 "managed"  : "True",
 and this needs to be changed to "managed" : "False".

Then it should be sufficient to check out the dev or stable branches 
("git checkout -b dev origin/dev", for example),
 run "gclient sync", maybe run "gclient runhooks" just to be sure, and then build.

Please reopen the issue if this doesn't solve all of your problems.

**Comment by zoechi** [comment]
I think a Dockerfile would be great. This would easily allow to reproduce build issues. What do you think?

**Comment by crunchex** [comment]
What solved my problem was following the old SVN-based instructions that I found here: https://groups.google.com/a/dartlang.org/d/msg/misc/FIoKBOD_Z_Q/w3OMvHw7YFkJ

I traced the issue back to the transition between 1.10.1 and 1.11.0, which I think around is when the dart organization made the switch to github. So with 1.10.1, those old instructions worked, and 1.11.0 is where the official instructions in this repo's wiki worked.

I can try the "managed" solution the next time I need to build from source. Not much experience with Docker, but I've heard great things about it.
